common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -814,6 +812,14 @@ div.problem {
         @include status-icon($gray-l4, "\f128");
       }
     }
+
+    &.inputtype.formulaequationinput {
```

Comment:
I had to squint funny at this code until I (think I) worked out what it was doing. Is the intention to make it not apply to ``.capa_inputtype.textline``? IMO it would be clearer to move it out one level to stop it accidentally applying where it isn't needed. If you think it is clearer as you have it then I'd add a comment explaining what it is doing.